### ISSUE TITLE
bug: owner can arbitrarily choose winner

### ISSUE THREAD

**Issue by patrick38894** [firelotto/firelotto#1]
owner (drawer) can end the game choosing whatever bitcoin block hash they want, allowing them to precompute and choose an arbitrary winner. instead choose a block number in advance (in solidity) and read the blockhash from an oracle (after ticket purchases have ended).

**Comment by polovaikin** [comment]
We are currently debugging the process which will only use the hash of the last bitcoin block that has been mined on the drawing beginnings, as it is described in our Whitepaper. This will solve the problem mentioned above.

**Comment by godsslave** [comment]
Thanks for your comment Patrick. Is this still true what you have found? Or it is already sorted out. Obviously I would be a bit reluctant to invest into FireLotto if they can choose who will be the winner.

**Comment by godsslave** [comment]
I spoke with Mikhail at the Ice at the Excel Arena. He is the CEO of FireLotto. I did share with him this issue. He said, this will be sorted out within days. I hope so. I think, this is a crucial issue, this is what differentiate a legitimate project from a scam.

**Comment by blam579** [comment]
This is a pretty big deal. Will be looking to see this addressed.

**Comment by godsslave** [comment]
Thanks, please.

**Comment by FIRELOTTOisAlsoCASHPOKERPRO** [comment]
They can change the time when the Bitcoin block is selected, but they can not influence the mining of this block. 


........ Which is what you would expect as they have to select a time for the lotto to be drawn and I can't imagine how they could ever influence the mining of it. As wouldn't that take away everything that makes Bitcoin secure being a transparent ledger? 

Anyway, that's the situation. I told them it was important to reply to this which they were going to tend to all questions in a couple of days but I asked for this to be addressed immediately so that people asking this question can decide to join the sale that's on currently.   

This is a super exciting ICO. It's going to make some very rich people.  

Good luck guys.

**Comment by godsslave** [comment]
They have to sort this out asap. Patrick is not happy, who found this out. You know edl2016 I am not a programmer, but Patrick is a programmer.

**Comment by blam579** [comment]
The lack of urgency and transparency of this team makes me question their conviction in providing a proper contender in this space. Sorry but it seems that the project itself is a lottery. Don't have time to wait on this project. Moving on!

**Comment by FIRELOTTOisAlsoCASHPOKERPRO** [comment]
Sorry but they have said it is not a bug. How is it possible for anyone to
mine the winning tickets from the blockchain? Is this really something that
can or could be possible to do? If it is then you will find that firelotto
are not aware of what you are suggesting. Here are their responses to me in
regards to this......

Please see photos of conversation.

**Comment by edl2016** [comment]
Sorry those photos were not showing so much....

**Comment by godsslave** [comment]
I cannot see any photo anywhere. Anyway , this is very important issue.

**Comment by edl2016** [comment]
I'm replying to you from email coming to my Gmail phone.

I will copy and paste here and supply pics later once home. Please just
understand I am nobody but an investor asking your questions and pushing
for an answer.


 Buck Shuk Al Laka, [12.02.18 05:45]
Hi there my research partner sent me this link about the final potential
issue we know of before recommending this to our group of investors and
before we buy a ton more.  I am wondering if you will be answering it or if
it was resolved? I don't post this in chat because I don't want to cause fud

Buck Shuk Al Laka, [12.02.18 05:45]
https://github.com/firelotto/firelotto/issues/1

Buck Shuk Al Laka, [12.02.18 05:46]
Is that a real problem? Is it being fixed or what's the deal with this? I
already have 16k flot and going to buy another bunch.  I support you 109%
and don't believe it's a scam

DanCo Rus, [12.02.18 05:47]
Hello

DanCo Rus, [12.02.18 05:47]
We are not the SCAM.

DanCo Rus, [12.02.18 05:48]
Will make update on GitHub within two days

Buck Shuk Al Laka, [12.02.18 05:49]
I'm just asking you if that is going to be an ongoing problem.  I don't
care about the problem myself but I can't get all my noobs in if it is but
I'm still buying in as I love your ico.   Can you just tell me personally
please if the issue will be ongoing? I showed you respect by coming to
private message about it.  Please show me the same respect

DanCo Rus, [12.02.18 05:50]
We don't have such problems

Buck Shuk Al Laka, [12.02.18 05:50]
So what they are saying is not accurate?

DanCo Rus, [12.02.18 05:51]
[In reply to Buck Shuk Al Laka]
Yes

DanCo Rus, [12.02.18 05:51]
Better write it in the group

DanCo Rus, [12.02.18 05:52]
[In reply to Buck Shuk Al Laka]
Thank you

Buck Shuk Al Laka, [12.02.18 05:52]
No I don't think many people know about it

Buck Shuk Al Laka, [12.02.18 05:52]
It's best to not cause alarm to the ones that don't

Buck Shuk Al Laka, [12.02.18 05:52]
Otherwise you'll have hundreds of people asking you hundreds of the same
question.

DanCo Rus, [12.02.18 05:57]
[In reply to Buck Shuk Al Laka]
You are right. Will make video soon with answers for the popular questions




Buck Shuk Al Laka, [12.02.18 06:08]
Ghost the other admin said this doesn't exist.  I don't ask in chat because
I don't want to cause FUD as I am a big time supporter of fire lotto and I
run a group of investors all buying in too.   So I don't want to cause
lies.   But he said this isn't a issue.   If it's not an issue why don't
you reply straight away like you replied to the last person who showed a
concern??

Buck Shuk Al Laka, [12.02.18 06:08]
https://github.com/firelotto/firelotto/issues/1

Buck Shuk Al Laka, [12.02.18 06:08]
Not causing trouble.  Just asking me to you

Ghost Rider, [12.02.18 06:12]
Just few minets

Buck Shuk Al Laka, [12.02.18 06:12]
People on Reddit are saying scam because of this and I have told them all
to shut up and give you time to respond like you did to the other maths
equation question your peers sent you.   That was great you responded but
for an investor we look for flaws and then try to cross them off as solved.
That tells us how much we can invest and if we tell our group.   So I have
to ask why are you waiting till after the ico to answer this as it's
important.

Buck Shuk Al Laka, [12.02.18 06:12]
Yes that's fine.   In your own time

Buck Shuk Al Laka, [12.02.18 06:12]
Just want you to know I'm not causing problems

Ghost Rider, [12.02.18 06:16]
Thank you for your feedback. Of course it`s impossible to change algorithm.
Our platform is absolitly open for any test

Buck Shuk Al Laka, [12.02.18 06:17]
Yes but why don't you reply to that in the GitHub because literally
hundreds of people in Reddit are waiting for the answer

Buck Shuk Al Laka, [12.02.18 06:17]
You replied to the big long probability question and that made everyone
happy.   Replying to this will too

Buck Shuk Al Laka, [12.02.18 06:20]
Anyway I won't keep you.  Thanks for replying.  I hope you answer the
question so more people jump in as many are waiting

Ghost Rider, [12.02.18 06:25]
Thanks, don`t worry!

Ghost Rider, [12.02.18 06:26]
[In reply to Buck Shuk Al Laka]
We can change the time when the bitcoin block is selected, but we can not
influence the mining of this block

Buck Shuk Al Laka, [12.02.18 06:27]
Thanks.

**Comment by godsslave** [comment]
I got that however this was the original response to Patrick's comment. We are currently debugging the process which will only use the hash of the last bitcoin block that has been mined on the drawing beginnings, as it is described in our Whitepaper. This will solve the problem mentioned above.
You got that. So since then there was no official response to this thing. They need to sort this out here. I don't think that this Patrick guy would be so silly. If they can make him happy that it is already sorted, then it is fine. This is an important issue and we fight with the time since the Ico will finish soon as well. If they would have sort this out quick, I would have invested already very very heavy into this for 10 ether or more. This is quite unfortunate this whole issue about this.

**Comment by FIRELOTTOisAlsoCASHPOKERPRO** [comment]
I believe there might be a misunderstanding with language or something so I
will give your post to the admin and see if they can follow this up.  If I
was anyone important I'd get it sorted but I'm not anymore than an
investor.

**Comment by patrick38894** [comment]
Hey I'm replying from my email because I'm on my phone. I originally just
posted the issue because I was reading random contracts from etherscan and
noticed it as a potential vulnerability that I wanted to notify people
about. I did not realize it would blow up  so much. Ihave no stake in this
at all and Im not mad or anything. Just wanted to bring up that the
contract wasn't trustless. If anybody wants suggestions on solutions I'd be
happy to brainstorm with you guys. I am always interested in collaborating
on solidity projects.

**Comment by godsslave** [comment]
Ok Patrick, if you are telling me, now it is sorted, then I will believe you. Yes edl, get an official answer from firelotto, I mean they should respond to this here so if Patrick is happy I am happy:) He is my eye:)

**Comment by patrick38894** [comment]
To be clear I'm saying I don't care. That does not mean the issue has been
fixed

**Comment by godsslave** [comment]
Hopefully, they will fix it soon. I believe they intend to fix this issue. Can you confirm if they have fixed it? So far nothing has happened if I understand correctly the situation. Well Patrick, clearly you have a big impact on the investors with your comment.

**Comment by polovaikin** [comment]
Hello.

We are currently using a random Bitcoin block which was mined after the drawing. This does not enable us to choose numbers, but it raises questions as to why this particular block was chosen 
We are at the final stage of debugging the system, after which the last block that was mined before the drawing will be used. Anyone will be able to check its hash on blockchain.info and make sure 
The lottery webpage will also have a user-friendly interface for converting the Bitcoin hash to the winning numbers in accordance with the RNG algorithm described in our white paper.

This means that we will solve the issue of transparency in the next few days.

**Comment by godsslave** [comment]
Thanks Polovaikin, hopefully you sort this out before your Ico finishes.

**Comment by edl2016** [comment]
Polo, I have spent so much time on your Reddit defending your business as a
contributing member who is also part of the discussions of the 25k buy.
I have been defending your business and putting things straight for days on
end, many hours , you can see in the attached images the amount of pages on
Reddit that I write explaining things when people are storming around
abusing firelotto.
I also run a little group full of noobs that I do for free where I spend 20
hours a day researching ICOs and giving our best of the month picks and
supplying our research.  I picked firelotto and told them all, 18 of them
that invested, that firelotto was not only the best of the month but it may
be the best pick we have had.
We are in talks with a massive bulk purchase to get your 32 cent pricing ,
plus we all have flot in our dashboards.

I sit there on Reddit fighting for you guys and explaining why the rumors
are there
And as you can see the time spent is very long.  My username is
330kbtctaxpaid.

Below are images of the messages I've been leaving for you and the
moderator of firelotto asked me to join him as a moderator as I have such
passion for your brand.

I entered the chat today and someone asked baiut the payout so I gave the
example of 32 million being a 3.2% yield and 10% of purchases makes that
happen and to go to page 11 on the Whitepaper.
Your moderator then banned me and was extremoey rude to me.  He told me
that yesterday he asked me not to post my refferal link so I said to him
that I would not do it again I then spent an additional four hours in
telegram helping out while no moderators were there answering questions
from your whiteppaer I know inside out which is why I sent my group to
firelotto and why I asked hacked to start a review on your ico.
He doesn't care though and he has banned me calling me a scammer

I can't believe this. First this moderator is telling people there is a
1200% profit.  Then I private a different moderator is telling me that he
was wrong and yes it is such and such.  The second I mentioned it I was
banned. You can go into your telegram and look up user name Buck Shuk alaka
and you will see all my contribution towards your telegram and towards your
firelotto.

I think this is pathetic and I think your moderators are way out of line
and I am writing this email to you to ask you kindly refund my firelotto
tokens and the rest of my group will also like this to happen but they can
email you on their own . I can't run a group of newbies who follow my
research and my picks for me to make firelotto the best pick ever, over
icon, over lalworld, over ethlend, over simple token and so on, I can't run
this group as firelotto being the big one while you are removing me from
your channel of information.  You will see that I have not abused anyone or
being manic or caused trouble or said a out of line word to your moderators
not even after ban.

So seeing you will not allow me to read the information from your ico I
need my money back from my purchase because I can't follow anything and
will have to hear things second hand.   I can't be a contributor or
investor when you shit out the information from me.  Laws in legal
investments say all investors must have access to the same information at
the same time.  I now don't have any of that.

Photos are attached of my contributions and my screenshot when I was banned
with your moderator.

Please reply letting me know what you need from me to refund my
contribution and let me know what email my group members should be going to
for the same.

Thank you.

Gary.
On Feb 13, 2018 06:20, "polovaikin" <notifications@github.com> wrote:

Hello.

We are currently using a random Bitcoin block which was mined after the
drawing. This does not enable us to choose numbers, but it raises questions
as to why this particular block was chosen
We are at the final stage of debugging the system, after which the last
block that was mined before the drawing will be used. Anyone will be able
to check its hash on blockchain.info and make sure
The lottery webpage will also have a user-friendly interface for converting
the Bitcoin hash to the winning numbers in accordance with the RNG
algorithm described in our white paper.

This means that we will solve the issue of transparency in the next few
days.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/firelotto/firelotto/issues/1#issuecomment-365032308>,
or mute the thread

**Comment by edl2016** [comment]
Dmitri, Roman, your admin at telegram is refusing me to speak to anyone
about a refund after banning me for being a good guy defending you guys and
has told me that I can sell my tokens at an exchange after the ico.

I can't believe I ask for a refund and he is telling me about exchanges and
that's what I have to do.

He has removed me from official channels of information and told me to sell
them on the exchange.

This is not professional.   You can't ban someone from your office new
releases channel and tell me you are keeping my money and I have to sell on
an exchange.  IVE NEVER SOLD A ICO TOKEN IN MY LIFE BECAUSE THE DAY I DO I
HAVE TO PAY TAXES ON EVERY EVENT.
and you are forcing me to do this.  That's not an option for me.

Please refund me from your ico.

I will next be writing to the lotto commissioner you are photos with
explaining to them why I have a comolaint to make and to tell them I have
gone through all other channels but am banned from them all over nothing as
you can read first hand in the screenshots I sent you or the chat history
at telegram.   That's the only option I'm being left with. My entire group
of buyers that have contributed to your ico need to speak to someone as
well about refunding because they are not comfortable having me denied
access to your information channel when it's me who updates them and keep
them calm during crashes and explains their concersn.  Please reply to me
ASAP as I will await till your pre ico is finished before I take serious
action as I don't want to hurt those investors in your ico which I probably
would not be able to if I tried but in the off chance I do cause an issues
I don't want to do that while others have money in your ico.

Hopefully you have similar respect and can write to my email or ask Roman
to.

I have stood by you guys more than anything.  I backed you guys up and
spent days in Reddit replying to people yelling scam and now you block me
over giving a refferal link to your ico to someone in their private message
with me explaining what I do and that I'd like them to come and see it and
in return for my time and effort of research and sharing it I ask them to
use my refferal link.  It was one person.  I was warned about it and I said
I won't do it again.  Then the following day I wrote your Whitepaper and am
removed.

Please address this.  Preferably in private but this I my only way to
contact you as I'm blocked all others.

**Comment by godsslave** [comment]
Thanks Edl for the info. After this I won't invest into Firelotto, however I did talk with the CEO personally and he was very nice and polite. I have problem only with the communication of its company. 
Also at the moment we are experiencing a bear market. Firelotto's price will probably drop after the Ico. Until they address this concern what we have here I wouldn't buy into firelotto even for of course 0.001$.
However nowadays a price drop after Ico is quite normal. Check out what have happened with we power? After the Ico, now it is on the market. 50% price drop. That is we power. Completely legit business and the ico was sold out within a few hours and so on. Anyway crypto will be fine long term, no doubt about that.

**Comment by edl2016** [comment]
WePower on many sites are fake tokens.   Go to the arcblock address and
enter it into ethdelta and the see the sales and follow the transaction
from a sale in trade history.  See the wallet in etherscan and follow where
it came from.  Now where it's come from is a guy who has made 100 million
fake tokens of wepower and arcblock and a few others and has listed them in
three different exchanges.   So maybe the price is down for this or maybe
it's those you are seeing that are down.

Personally I'm not going to be that guy bad mouths firelotto because they
banned me.. as far as I'm concerned they banned someone who worked their
ass off in support of them to the point I was asked to moderate a firelotto
Reddit page.
Many ICOs go through issues and you hear lots of stories like mine but
don't for one second think that these stories will ever touch the calue of
the ICOs when they hit market because they don't.
Firelotto as much as I don't want to talk it up and just want to abuse
them, but their lotto is going to be popular, they will work their kind out
the guys involved are professionals and trying to do their best with a new
technology and it's the start of crypto so we can't exoerct them to get
everything right as there aren't all that many experiences experts for all
the companies that want to join the space.   But these guys have
processions and reputations they probably care about and some must be very
intelligent.

I wish I could bad mouth these guys but the truth is they probably will be
as profitable as everyone thinks they could be but they need to burn
tokens.  You see if they sell 16 million tokens for 2 bucks each that's 32
million dollars and split between tolens it works out to 3.2 cents per
token per year. So you aren't getting alot for your investment but then you
have to add that this is crypto and everything is thousand times bigger
than you ever expect so maybe it is more but it's a lot and it's going to
be difficult maybe or it's going to be simple like flies to these
moderators nappies.

I'd like a refund along with my entire groups that follow me just to show
them the finger that they really did do the wrong thing to the wrong guy,
but sometimes there's and ico that you just have to have even a tiny piece
off because they become so expensive that buying like 200 icon now is like
800 bucks.  I got tens of thousands for 2800. I could never buy them at
exchange so having a bit of this I can live with as they will never refund
me and I could never abuse them enough for them hear because of all the
idiots that do abuse at the first step.

I will tell you something , that I love the support you just gave me, so
send me your public wallet address or send me a new one that you don't care
is online for a few days and I'll send you 0.3 eth of tokens.  It's not
alot but at least you'll have them just in case.

Seriously, I'd love to.   I don't care for them and I have more than enough
but am just glad this happened now because we were in agreement for 25k
worth of them for a cheaper price.  Not much cheaper but whatever.
Glad I'm out of that now.

And honestly I now hope they fail die to the lack of respect.  Even when I
said I got asks to be moderator the admin guys says we don't need
anyone.....look at the Reddit you dumb mutha ffff.

Peace.

**Comment by godsslave** [comment]
Fake tokens? On behalf of Wepower? If that is true, then wepower is in deep shit anyway. Well I go to cryptomarket.com and I go to wepower and I see the price which is now even lover than 50% compare to the original price. So that price looks pretty real. Pretty much the real wepower.
I love the concept of Firelotto but I don't love when they don't address an issue which is that serious like this such as company can select who will get the Jackpot. 
Sure, Firelotto will be fine as long as they can prove that it is a legit project and not a scam. Ask Patrick who has started this whole thing. 
Sure, 0.3 eth would be fine. It is still quite a lot.
However hold on. What support I gave you? Not much.
But you are really kind. 
 I was just concerned about the firelotto project whether it is legit or not. I want to invest into something real. I know that casino's tend to be nasty many times. I have too much experience about casinos myself. 
Really, you got an agreement with Firelotto for a cheaper token price?

**Comment by DanCoRus** [comment]
Dear edl2016, you made promotion in official Fire Lotto Telegram group for your referral link and your own group. Also you sent PM to the community members. Had more than 10 complaints from different community members about your promotion. Please write in current topic your questions or opinion about source code.

**Comment by DanCoRus** [comment]
Godsslave, Fire Lotto didn`t have any agreements with edl2016 (in Telegram as Buck Shuk Al Laka).

**Comment by edl2016** [comment]
Well that's a lie isn't it Dan because I have photos of our conversation
where you said you banned me because I gave a refferal link. Which I
apologized for and then the next day you banned me.  I asked why you said
that.  Then you said I made a promotion for my telegram group....but I
don't have a telegram group....I have my housekeeper my gardener and my
truck driver and couple other friends on text message and someone asked me
what I do or something and I explained that's what I do.  They asked me
because half my group was in your telegram buying firelotto.

You now have just said that you had 10 complaints about me.  But in the PMs
that I scrwenshotted it shows you saying you had 5 complaints.   How come
it is 10 now?
You kicked me out because I corrected you when you told someone they
receive 10% tokens from the lotto and I relied saying actually guys make
sure you realise it's about 3.2C per token per year according to page 11 of
the whiteppaer which you told me to read.  Then instantly I was banned.
First my post then me.

You are flat out lying.  But I don't care anymore because if that's the
person you hold yourself up to be and you feel proud of yourself then
that's great. I wish you all the power in the world with your employment as
telegram moderator.  Hopefully that's you achieving your goals. I don't
have any opinions about your source code. I never had except everyone here
and everyone in Reddit will always tell you all of have done is support you
and stand up for you .
It now makes sense why you banned me though now you mention that.   It's
blatently obvious you banned me because I posted your statements here in
this GitHub the other day.   Next thing I know you banned me.   So that's
why? Did you read the comment where you tolde I should tell the whole
group? So why now do you have a problem.   I was showing these people that
you addressed the issue and Sid there was no problem....oh but hey, I get
it now....were you lying and you are barrased that I posted it here and you
were lying? Is that why you banned me? Maybe you should read what I write
instead because it was standing up for you.    My username is
300kbtctaxpaid on Reddit and you will see all my posts of me standing up
for you.

Didn't you see the photos and screenshots I sent here? So I need to log in
on desktop to post do I? Ok . JUST MAKE SURE YOU K OW I don't have a
promotion. If I do can you please show me it here? Can you show me my
promotion please. You showed me a screen shot earlier of me saying that I
said I had a group of noobs that I share my research with.  Which I do.
And I do for free as they are my friends.


AS FOR THE OTHER POSTER Godsomeone, no please, I offered you 0.3 eth of
tokens, I will stand by my word.   I will admit thugh it is getting very
very difficult not to start abusing this bloke who obviously doesn't read
how much great stuff I write about them in Reddit.  Maybe someone could go
to Reddit look up my username and post what I write about them as I can't
get to a desktop to upload screenshots properly.  Id love someone to prove
that I've said nothing but good things in their defense.   Actually to
prove my innocence as it is now at question, I will give that person 1 eth
of these lotto tokens.  Whover does that.   The second they are released I
will give 1 eth worth.  Unless of course these guys refund me and then I'll
give you 1 eth.

Anyway this moderator is on some power Trip but now I think it's cause I
posted his comments.  Anyone notice how then he wants nice to me and now
he's like this.   He told me to post them.  I don't get it.  Did the
screenshots I send show up here because I have screenshots of my
conversation with him.
I'm sire he's find a way to change whatever he wrote, and no doubt what I
wrote, but that's me thinking more I to this than it is.


Anyway I can't believe they try to shame me here when I'm speaking so good
about them.  It's strange.

**Comment by godsslave** [comment]
Well, it would have been not unheard of when an ICO does have a preferential treatment for a large investor. This is completely normal in the Ico world or in general in the business world. Thanks for the info anyway. My main concern is this issue what we have at hand. I think for firelotto the lesson is to take seriously github issues. Nowadays investors do check github even though I am not a programmer.

**Comment by edl2016** [comment]
No just in private message after I posted your words here as you asked me
to and now you like about the complaints you had about me.  The number
changed from me being banned for sending a person a pm in telegram of my
referral code which was nothing to do with you and then it becomes me
advertising my friends that I help as some promotion and then from 5
complaints to 10 complaints.   Let's be real mate, you are full of lies.

I'm going to go to desktop in minute and post all our communication before
you get a chance to change it all and post it.  You're a flat out liar.

**Comment by edl2016** [comment]
1.3 eth worth of firelotto  to you I swear on my baby son's life.  If you
go to Reddit and show all my writings about firelotto here as this guy
needs to see he's a fool for going after his biggest supporter.

**Comment by edl2016** [comment]
For the record I sent one on to one member.  I can probably get a copy of
all my telegram activity if someone wants to tell me how.

**Comment by edl2016** [comment]
Dan, I've been back in your telegram group for the past few hours.   I have
a different name of course but you can just talk to me there if you like.

**Comment by edl2016** [comment]
https://photos.app.goo.gl/q5pzD5bvqthC6x3y1


Screenshots of how he banned me and why he banned me and his response to
everything.   I was then cut off with nothing further until here in this
GitHub. I've tried to remain professional but he is already lying about 10
complaints and me making promotions.  Everyone knows I don't promotoe
anything or surely I'd be promoting it in Reddit. Truth be told I did do a
thread earlier today saying I would start a telegram group.  That's great
timing.

Anyway I'm done here. Nothing more to say.

The 0.3 eth is there for you if you send me your wallet and 1 eth still on
offer though if it's not already sorted then leave it as this shows the
screenshots.

**Comment by godsslave** [comment]
Relax man. Everything is cool. I believe you, I am sure you have defended firelotto in Reddit. I did also promote and defended Firelotto, however this issue is a big issue what we are talking about. Would you send me your firelotto tokens? You are nice, you want to make me a millionaire?
However first firelotto needs to deal with this issue what we have at hand and I am only concerned about the speed how they have treated this issue so far. This issue was posted by Patrick 22 days ago. 22 days in crypto world is like history already.

**Comment by edl2016** [comment]
Yeah wow 22 days is a long time.  I missed that part as only saw a more
recent date and didn't think to look back.
I'll leave you guys to sort that out as I have no reason to relay any
messages on any further.

I offered you firelotto tokens, my word is my bond, it's all a man has in
the reality I live in.  So if you send your ethereum wallet for them to be
deposited into that will happen. I have nothing more to say about these
people as it will just break my character what I'd like to tell this person
that's making lies about me.

So this thread will be considered spam in my email account as I'm done
ewith this ico.

To send those tokens please send your wallet through Reddit as a pm.


Cheers pal and yes I thought he said it was solved and turns out it was a
lie so I have no interest in  a liar and am leaving before I say something
that will hurt my charachter.


Thanks pal.

**Comment by DanCoRus** [comment]
eld2016, I never talked with you about any discount for the Fire Lotto tokens. You know my username in Telegram, but you send screnshots from another chat with another person (any person in telegram can use my nickname and my photo, but can not use username). 
If you have any questions about Fire Lotto, please send them to info@firelotto.com  Fire Lotto admins NEVER discuss any discounts in Telegram and NEVER send any PM messages with wallet numbers.

**Comment by DanCoRus** [comment]
Goldsslave, eld2016 try to CHEAT and SCAM you and another users. He can not send any tokens to you. 
Till the end of ICO investors can not move tokens from investors account.
In the end of ICO investors will have function to add ERC-20 Ethereum wallet and move tokens to it.

**Comment by godsslave** [comment]
Alright, but how on earth would he be able to scam me or any other guy with a public Ether address?

**Comment by DanCoRus** [comment]
Dear community please be carefull with eld2016 as Buck Shuk Al Laka in Telegram messenger. Also he have another nicknames and accounts.
I am ADMIN in official Fire Lotto Telegram group.
Fire Lotto admins NEVER discuss any discounts in Telegram messenger and NEVER send any PM messages with wallet numbers.

**Comment by DanCoRus** [comment]
Godsslave, he will send to you invitation to the special group with SUPER prices for Fire Lotto tokens or 40-50% discount for the tokens in PM. Please don`t trust such persons and messages.
Every dau ADMINs BAN 5-6 scam admins in official Telegram group.
You can buy tokens JUST in investors account on firelotto.io

**Comment by godsslave** [comment]
Ok, that is obvious. If someone would offer me fire lotto token with 50% other than firelotto that is a scammer. There are plenty of scammer around for sure, but to send me firelotto token sounds nice however technically it is not possible at the moment:) ICO will finish within 2 days?

**Comment by edl2016** [comment]
i NEVER SAID YOU OFFERED ME ANY DISCOUNTS.

**Comment by edl2016** [comment]
who am i trying to cheat and scam? i made it clear i would send him the
tokens when they were able to be moved. it says it there in my writings.
why are you trying to make me look like a scammer?

**Comment by godsslave** [comment]
It is true, probably this whole thing is a communicational problem. I have misunderstood what edl said about the 25k.

**Comment by edl2016** [comment]
seriously where and what is the scam? all i did was offer someone some
tokens. how am i scamming them or cheating them? I said they could have
them once they were able to be moved. why would a moderator be here trying
to tell people i am a scammer because i offered someone 0,3 tokens? this is
seriously very immature. if this is how you deal with complaints you will
never get a license by making these unfounded accusations

**Comment by edl2016** [comment]
i have never said you offered any discounts. i said that we (my research
partner) was offered 25 eth at the price of a 50eth package which would
take the price from 38c per token to 32c per token. i can provide
screenshots of this if you like. it was not with me but with my research
partner

**Comment by edl2016** [comment]
why are you making these lies about me up? now i can safely say that you
are defamating my name and this is not accpeptable. i have never in my ife
scammed anyone and i write all this from my original and long time email
address that is linked to every sinfgle ico i have ever been part of. If
you do not cease with this defamation I will be forced to take a lawsuit
out against your company. if this continues i will take legal action
against the firelotto company, this i guarantee. i have never ever in my
life scammed anyine and people know that y reading my history in reddit and
that i have been defending you and not you are making all these absurd
accusation which i will not stand for.

**Comment by edl2016** [comment]
i said i would send someone 0.3 eth worth of firelotto tokens and i will
stick to my word. you have nothing that suggest otherwise., as soon as they
are unlocked this will happen. what you are doing is outrageous and far
beyond acceptable. I will be waking up in the morning and having my
barrister take legal action against your company. this is not a threat, it
is exactly what will happen and you should want to make sure you CEO knows
this.

**Comment by edl2016** [comment]
Im sorry but what have you misunderstood? here is a screenshot. this attack
is unwarranted. please see this screenshot of what i was talking about as
to the 25.
https://photos.app.goo.gl/AjJTxNDd1mCIIde53

**Comment by edl2016** [comment]
if anyone wants to call me a scammer , calli to me personally. my phone number is +61455755558 and i will be here waiting.

**Comment by edl2016** [comment]
my telegram name and my reddit name are things i take with great pride. i have never ever lied or tried to scam anyone. i am a good person and this slander and defamation must stop. if i see anymore of it from this profesisonal company that i have sent ethereum to, i will spend every single last crypto cent i have on suiing this company. I will be forwarding this github to the CEO of firelotto and I will be showing the licenseing comission that this is how they handle someone who spoke decently about them. where have i ever said a bad word about firelotto? Though is it not amazing that this companmy takes 22 days to reply to a question but here they take seconds to attack a investor that they are going after for no reason

**Comment by edl2016** [comment]
dan how about you give me a phone number so i can contact you right now. or are you really a moderator of a telegram of a legitimate company? i am sendinglinkedin requests to the main peopl eof the firelotto team to show them this nonsense and this defamation their company is attacking my charachter with. this is absurd that a legit company can be doing this when there is anot a single thing where i have said anything bad about this company. this is absurd and i will be taking legal action if this continues  send me your phone number or call me on mine +61455755558 NOW

**Comment by edl2016** [comment]
well if you are not going to give me your number or call me then i am forced to contact thos members of your team such as Alexander Zasypkin at hi office where he has his day job because i wont stand for this. not ever. thankfully their contact details are everywhere.

**Comment by edl2016** [comment]
You can tell roman demidov that he will be expecting a lawsuit filed in Cyrpres in the coming weeks. This is not something I can stand by and allow to be said about me. I am forwarding everything here to the lottery licensing that you were photographed with and apparently on the way to get your icense wiht themm. No way will they stand for their licensees to behave ni nthis manner. Not a single bad word has been said by myself towards firelotto and i have been attacked all because i tried to help people get past this patrick question. All my posts in reddit have been standing up for reddit. this is obsurd and i am deeply hurt such an attack is taking pace and nobody has the courage to phone me or to show prrof of what they are saying yet i am showing all the proof in my screenshots and my links with my phone number. outrageous

**Comment by godsslave** [comment]
Relax, I don't call you a scammer. It was probably a misunderstanding this whole issue. I appreciate your transparency. Probably DAncorus referred to real scammers and not to you. For example when I wanted to buy into Vestarin at pre ico , many many scammers has attacked me at Telegram. Well when there is an ico launch it is not wise I think to go to Telegram for sure:).

**Comment by edl2016** [comment]
He is saying that I am  a scammer and I will scam you and everything. he is making up lies about me and is going beyond what any reasonable person would do to someone who has only been talking good about this company. Someone please proovide me with a single bad word towards this company. I have the scrrenshots everywhere but this is outrageous. I have now sent a email to every single company under their linkedin profiles and to the every email that is registered to the domain firelotto.com so hoping someone with soe decency listed on their team will reply. I have contacted their instagrma and twitter accounts of the team and i hope one of them areply because no way is this guy part of their team or anymore than a hired telegram hand. No way can be left to do this to me online when everything about me is legitimate from my email to my phone muber to my intials of my name. I will not stand for that and  my innocence is proven if anyone looks at this single link here with everything in it that i have ever said....
here is a link to the images showing my interactions with or about firelotto and there is a image of the scr4eenshot of a deal being negotiated with my research partner and firelotto in relations to the nuber quoted of 25eth.
someone pease tell me what i have done wrong or what or where i have tried to scam. and godsslave, yes you can have my 0.3 eth worth as promised. at this rate and the way im feeling you can probably have it all because i am disgusted by this. https://photos.app.goo.gl/WuzFodPtOXlnA4jf2

**Comment by edl2016** [comment]
godsslave, here is what he the admn at telegram wrote about me to you,. your message is copy pasted there to just cause i couldnt seperate. please see what hes saying about me. this is unacceptable wiuth no basis/

Goldsslave, eld2016 try to CHEAT and SCAM you and another users. He can not send any tokens to you.
Till the end of ICO investors can not move tokens from investors account.
In the end of ICO investors will have function to add ERC-20 Ethereum wallet and move tokens to it.

 @godsslave
 
godsslave commented an hour ago
Alright, but how on earth would he be able to scam me or any other guy with a public Ether address?

 @DanCoRus
 
DanCoRus commented an hour ago
Dear community please be carefull with eld2016 as Buck Shuk Al Laka in Telegram messenger. Also he have another nicknames and accounts.
I am ADMIN in official Fire Lotto Telegram group.
Fire Lotto admins NEVER discuss any discounts in Telegram messenger and NEVER send any PM messages with wallet numbers.

 @DanCoRus
 
DanCoRus commented an hour ago
Godsslave, he will send to you invitation to the special group with SUPER prices for Fire Lotto tokens or 40-50% discount for the tokens in PM. Please don`t trust such persons and messages.
Every dau ADMINs BAN 5-6 scam admins in official Telegram group.
You can buy tokens JUST in investors account on firelotto.io

**Comment by edl2016** [comment]
Wow now I am digging I am finding some crazy stuff. There is a lawyer in Russia with the same name as firelotto lawyer and every single image of this lawyer in google images with the same things in the CV are are all a different person. There is only one single photo of this lawyer and it is the same photo used in both ICOS that she has done with the same members of the firelotto team. The firelotto team did a ICO called Cash Poker Pro, yet i can not find any photo with this lawyer other than this one on their webpage and the other lawyer, an older woman, she has the same research in her CV as listed on this lawyers bio on firelotto website. 

CASH POKER PRO ICO FINISHED ON 18 NOVEMBER 2017 and with the same team members, they had a ICO price as of december 17th here are there results, though some websites have them at 85cents to buy in, but this is from december....
ICO price is 0.0018 ETH / CASH
Now price is 0.000096999  / CASH
Very very undervalue than ICO price
about -1755 % undervalue than ICO

https://icobench.com/ico/cash-poker-pro

as of january 9th this message was left from an investor....

Not ONE word from any rep since 1st of Dec, nor forums, tweeter etc... unfortunately it looks alot like a scam. I m so sorry I invested in this. ETH 4x since then, fml.
Russians and poker... should ve known better....

Febuary 10th this was left....
Thank you all for donating to Russian Orphan Fund. Спасибо.

at this link you will find the team members who were in on cash poker pro ICO that has not tweeted or mentioned anything since december 8th....
https://relationshipscience.com/person/dmitry-polovaikin-236991940

thye have been on coin market cap for 4 months now and their market cap is listed as 0. 
https://coinmarketcap.com/currencies/cash-poker-pro/

the only market they are on is etherdelta

Here is their website, same names different photos 

https://cashpokerpro.io/


Make what you want of that but its a bit quick to go from one ICO to another in 2 months without contacting their investors in 3 months isnt it? 

You guys decide but i am not going to bother wasting my money on a lawsuit with these guys who are obviously have questions to be answered.

**Comment by edl2016** [comment]
Check out what they write about the laws of cash poker pro and licenses, no doubt the same as this one.....
https://www.reddit.com/r/ethtrader/comments/793gjm/own_your_slice_of_the_next_big_thing_in_online/
Due to decentralized technology, Cash Poker Pro will be able to offer its services in any possible geography with absolutely no risks of getting banned.
https://www.reddit.com/r/ethtrader/comments/793gjm/own_your_slice_of_the_next_big_thing_in_online/

-----------
And here is what they say about lowering their overheads 

Lower operating costs

Licensing, banking, legal, and the actual costs of flipping an IPO are sky-high, especially in the iGaming industry. Cash Poker Pro solves this issue by eliminating most of the unneeded overheads and focusing on what truly matters - the best possible gaming experience and the industry low rakes.

-------------------


So it makes you wonder if them talking about their photo of them with the gaming license guy at a expo they visited actually has any merit if their other online gambling scheme is the same mentalioty as this, which it is the same team so why would it be different.

and they say im the scammer

**Comment by edl2016** [comment]
Well they really did stick with the same team from cash poker pro. You can see here that their telegram moderators are the same even https://bitcointalk.org/index.php?topic=2194237.0;prev_next=prev
Hi
We are looking for Telegram manager for our chat about Cash Poker Pro on Chinese
If u have any questions or offers, pls send PM on Telegram - @DanCoRus
TY.

**Comment by edl2016** [comment]
https://bitcointalk.org/index.php?action=profile;threads;u=1087329;sa=showPosts
wonder why the CFO is different on this versus their website.

**Comment by godsslave** [comment]
Thanks for your research. I knew about cashpokerpro. It was a failed project. But this is not an unheard thing. However they have to sort out this issue what we are talking about if they want to clear up their name, if that is possible at all at this point.

**Comment by polovaikin** [comment]
We have finished debugging the process of drawing. From now on, the Bitcoin block used for drawing numbers will be chosen according to strict rules. It will be the first block mined after the timer stops. The number of the block which was used for drawing numbers will be published on the lottery website and shown in the lottery archive. We have added the Transparency section, where one can read about the algorithm used in drawing numbers and fill in a special form with the Bitcoin hash to check past winning numbers. It means that everyone will be able to make sure that the lottery is transparent and the drawings are fair.
![img-2018-02-14-13-08-44](https://user-images.githubusercontent.com/1670469/36198882-eca90856-1188-11e8-9caa-b167eb514156.png)
![img-2018-02-14-13-09-28](https://user-images.githubusercontent.com/1670469/36198883-ecc61f7c-1188-11e8-8ffe-4fe5fccf9148.png)
![img-2018-02-14-13-10-12](https://user-images.githubusercontent.com/1670469/36198884-ece3694c-1188-11e8-87ee-084b6b1ece38.png)
![img-2018-02-14-13-11-26](https://user-images.githubusercontent.com/1670469/36198885-ed00a458-1188-11e8-8b6c-c8a5dd5e05a3.png)

**Comment by patrick38894** [comment]
well done!

**Comment by godsslave** [comment]
So this should be fine Patrick? Ok, then I invest.
